index.d.ts

```diff
@@ -1668,6 +1672,54 @@ declare namespace Moleculer {
 		}
 	}
 
+	/**
+	 * you can use multiple modifier with a dot
+	 * e.g. `black.bgRed.bold`
+	 */
+	type KleurColor = keyof Kleur | string;
+
+	type ActionLoggerOptions = {
+		logger?: LoggerInstance;
+		logLevel?: LogLevels;
+		logParams?: boolean;
+		logResponse?: boolean;
+		logMeta?: boolean;
+		folder?: string | null;
+		extension?: string;
+		colors?: {
+			request?: KleurColor;
+			response?: KleurColor;
+			error?: KleurColor;
+		};
+		whitelist?: Array<string>;
+	};
+	type TransitLoggerOptions = {
+		logger?: LoggerInstance;
+		logLevel?: LogLevels;
+		logPacketData?: boolean;
+		folder?: string | null;
+		extension?: string;
+		colors?: {
+			receive?: KleurColor;
+			send?: KleurColor;
+		};
+		packetFilter?: Array<Packets.packetType>;
```

Comment:
Two things here:
1) Since you've moved `PacketType` into its own type, you can just reference that without the leading `Packet.`
2) Please use `PacketType[]` instead of `Array<...>`

Reply:
PacketType is in the Packet namespace . and not the packetFilter, So I need to use the `Packet.PacketType` notation